packages/status-js/src/client/community/community.ts

```diff
@@ -129,24 +135,34 @@ export class Community {
 
         this.chats.set(chatUuid, chat)
 
-        this.client.waku.relay.addDecryptionKey(chat.symmetricKey, {
+        // this.client.waku.relay.addDecryptionKey(chat.symmetricKey, {
+        this.client.waku.filter.addDecryptionKey(chat.symmetricKey, {
           method: waku_message.DecryptionMethod.Symmetric,
           contentTopics: [contentTopic],
         })
 
+        const unsubscribe = await this.client.waku.filter.subscribe(
```

Comment:
todo: see stream limits and consider changing js-waku/go-waku's so one content topic can be unsubscribed from a stream serving many other topics without service interruption